app/src/androidTest/java/ch/hikemate/app/ui/saved/SavedHikesScreenTest.kt

```diff
@@ -193,35 +257,16 @@ class SavedHikesScreenTest : TestCase() {
             .assertCountEquals(3)
       }
 
-  @Test
-  fun errorIsDisplayedIfOneOccurs() =
-      runTest(timeout = 5.seconds) {
-        `when`(savedHikesRepository.loadSavedHikes()).thenThrow(RuntimeException("Error"))
-
-        composeTestRule.setContent { SavedHikesScreen(savedHikesViewModel, navigationActions) }
-
-        composeTestRule
-            .onNodeWithTag(CenteredErrorAction.TEST_TAG_CENTERED_ERROR_MESSAGE)
-            .assertIsDisplayed()
-        composeTestRule
-            .onNodeWithTag(CenteredErrorAction.TEST_TAG_CENTERED_ERROR_BUTTON)
-            .assertIsDisplayed()
-            .assertHasClickAction()
-            .performClick()
-
-        verify(savedHikesRepository, times(2)).loadSavedHikes()
-      }
-
```

Comment:
Question: Why did you remove that ?

Reply:
There is no error handling for now in the view model, so nothing happens if an error occurs in the repository, there is nothing to test, except that the app stays on the loading screen forever...

Reply:
Okay, I've seen there is a task for this in the PB. Sounds good